documentation/src/main/asciidoc/reference/search-dsl-highlighting.asciidoc

```diff
@@ -304,8 +304,9 @@ include::{sourcedir}/org/hibernate/search/documentation/search/highlighting/High
 
 [NOTE]
 ====
-These options are supported by all highlighter types on the Elasticsearch backend,
-but only by plain and fast-vector highlighters with the Lucene backend.
+These options are supported by all highlighter types on the Elasticsearch backend.
+As for the Lucene backend -- the number of fragments is also supported by all highlighter types,
+while only plain and fast-vector highlighters support fragment size.
```

Comment:
I have been thinking about doing so. Sadly those `BreakIterator`s won't help. We may try to use them for some Lucene-specific highlighter options. As for this fragment size option, we'd need something like this - https://github.com/elastic/elasticsearch/blob/main/server/src/main/java/org/elasticsearch/lucene/search/uhighlight/BoundedBreakIteratorScanner.java#L36 . And since that iterator is a bit lengthy I've kept it out 😃🙈